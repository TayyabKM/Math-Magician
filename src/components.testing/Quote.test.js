import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/calculatorPage/DisplayQuote';

test('Quote SnapShot test ', () => {
  const component = renderer.create(
    <Quote />,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
