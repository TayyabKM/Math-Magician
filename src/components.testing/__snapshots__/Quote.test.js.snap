// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote SnapShot test  1`] = `
<section
  className="quote_display"
>
  <h2>
    Quote
  </h2>
  <p>
    Loading quote...
  </p>
</section>
`;
